documentation/trunk/solutions/basic-computing/eiffelbase/eiffelbase-tutorial/eiffelbase-kernel.wiki

```diff
@@ -102,7 +102,6 @@ These two features enable clients to ascertain the dynamic type of an entity at
 
 ==Miscellaneous==
 
-The query Void, of type <eiffel>NONE</eiffel>, denotes a reference that is always void - not attached to any object. <br/>
```

Comment:
Actually `Void` is of type `detachable NONE`. This explains why it cannot be a value of an attached expression (detachable types do not conform to attached types). And `NONE` still conforms to all other types. It's just one cannot create an object of type `NONE`.